### PR TITLE
Update strings.po

### PR DESCRIPTION
**Pull request by djdirty60**
Updated

### Check if this PR fulfills these requirements:
* [x] My changes respect the [Kodi add-on development rules](https://kodi.wiki/view/Add-on_rules)
* [x] I have read the [**CONTRIBUTING**](../../master/Contributing.md) document.
* [x] I made sure there wasn't another [Pull Request](../../../pulls) opened for the same update/change
* [x] I have successfully tested my changes locally

#### Types of changes
- [x] New feature (non-breaking change which adds functionality)
- [ ] Feature change (non-breaking change which change behaviour of an existing functionality)
- [ ] Improvement (non-breaking change which improve functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

**Review by CastagnaIT — CHANGES_REQUESTED** [review]
Currently this translation is maintained by @michaelarnauts,
but i see has not been updated for some time
if the previous maintainer have no intention of continuing to keep it up to date, i might suggest a change of maintainer, if you wish of course

changes will be accepted upon confirmation of @michaelarnauts, If no reply will be received, they will be accepted as they are.

There are some things to be fixed:
- When you create a PR please write an appropriate Title
- Addon checker check has failed see check error details
- Some ID's are not filled, please add missing translations

File: resources/language/resource.language.nl_nl/strings.po [hunk 1]
```diff
@@ -327,7 +327,7 @@ msgstr "Onthoud geluid / ondertiteling voorkeuren"
 
 msgctxt "#30083"
 msgid "Force the display of forced subtitles only with the audio language set"
-msgstr ""
+msgstr "Forceer de weergave van geforceerde ondertitels alleen met de audiotaal ingesteld""
```
Comment (michaelarnauts) [inline resources/language/resource.language.nl_nl/strings.po:330]: Duplicate end quote

File: resources/language/resource.language.nl_nl/strings.po [hunk 2]
```diff
@@ -157,31 +157,31 @@ msgstr ""
...
 
 msgctxt "#30040"
 msgid "View ID for movies"
-msgstr ""
+msgstr "ID voor films bekijken"
 
 msgctxt "#30041"
 msgid "View ID for shows"
-msgstr ""
+msgstr "ID bekijken voor shows"
```
Comment (michaelarnauts) [inline resources/language/resource.language.nl_nl/strings.po:172]: "ID voor shows bekijken" so it is consistent?

File: resources/language/resource.language.nl_nl/strings.po [hunk 3]
```diff
@@ -68,11 +68,11 @@ msgstr "Alle soorten lijsten"
 # Unused 30011
 msgctxt "#30012"
 msgid "Profiles options and parental control"
-msgstr ""
+msgstr "Profiel opties en ouderlijk toezicht"
```
Comment (michaelarnauts) [inline resources/language/resource.language.nl_nl/strings.po:71]: Profielopties should be one word

**Comment by CastagnaIT** [comment]
If no reply is given within 3 days, this PR will be closed

**Pull request closed without merging**